proxy/proxy.go

```diff
@@ -244,7 +246,7 @@ func ParseUpstreamsConfigEx(upstreamConfig, bootstrapDNS []string, timeout time.
 func (p *Proxy) Init() {
 	if p.CacheEnabled {
 		log.Printf("DNS cache is enabled")
-		p.cache = &cache{cacheSize: p.CacheSizeBytes}
+		p.cache = &cache{cacheSize: p.CacheSizeBytes, cacheMinTTL: p.CacheMinTTL, cacheMaxTTL: p.CacheMaxTTL}
```

Comment:
just to make it look better - let's split this into multiple lines

Reply:
Done.